modules/mono/glue/GodotSharp/GodotSharp/Core/Vector2.cs

```diff
@@ -191,6 +191,23 @@ internal void Normalize()
             );
         }
 
+        /// <summary>
+        /// Returns a new vector with all components clamped between the
+        /// <paramref name="min"/> and <paramref name="max"/> using
+        /// <see cref="Mathf.Clamp(real_t, real_t, real_t)"/>.
+        /// </summary>
+        /// <param name="min">The minimum allowed value.</param>
+        /// <param name="max">The maximum allowed value.</param>
+        /// <returns>The vector with all components clamped.</returns>
+        public readonly Vector2 ClampF(real_t min, real_t max)
```

Comment:
I think we may want to use overloads in C# instead of a suffix.

```suggestion
        public readonly Vector2 Clamp(real_t min, real_t max)
```

Reply:
Good point missed the `PosMod` below which uses the same, think I took this from `MathF` having some methods, will update